Gemfile

```diff
@@ -5,12 +5,3 @@ gem 'rack'
 gem 'grape'
 gem 'cucumber'
 gem 'aruba'
```

Comment:
No, we still need it. Cucumber does not come with ChefDK. https://travis-ci.org/chef/delivery-cli/jobs/233230233#L383